pkg/cmd/pr/status/http.go

```diff
@@ -186,12 +187,16 @@ func pullRequestStatus(httpClient *http.Client, repo ghrepo.Interface, options r
 	return &payload, nil
 }
 
-func pullRequestFragment(httpClient *http.Client, hostname string) (string, error) {
+func pullRequestFragment(hostname string, showConflicts bool) (string, error) {
```

Comment:
this is private function made sense to change the signature to support this change